test/e2e/pattern.test.js

```diff
@@ -111,4 +111,21 @@ describe('vl-pattern', async () => {
     await assert.eventually.equal(input.getValue(), '1c6fa548-5eef-11eb-ae93-0242ac130002');
     await assert.eventually.isTrue(input.hasUuidPattern());
   });
+
+  it('als gebruiker kan ik alleen een numeriek patroon invullen', async () => {
+    const input = await vlPatternPage.getNumericalInput();
+    await assert.eventually.isTrue(input.hasNumericalPattern());
+    await assert.eventually.equal(input.getNumericalDecimalScale(), "4");
+
+    await input.setValue('foobar');
+    await assert.eventually.equal(input.getValue(), '');
+    await input.setValue('123foobar');
+    await assert.eventually.equal(input.getValue(), '123');
+    await input.setValue('1234567,8901');
+    await assert.eventually.equal(input.getValue(), '1 234 567,8901');
+    await input.setValue('-1234567,8901');
+    await assert.eventually.equal(input.getValue(), '-1 234 567,8901');
+    await input.setValue('1234567,89012');
+    await assert.eventually.equal(input.getValue(), '1 234 567,8901');
```

Comment:
Dit betekent dat dat automatisch zo geformatteerd wordt? Want ik persoonlijk vind die spaties als thousand separators schijte-vervelend. Maar als dat ook weer als huisstijl moet/kan beschouwd worden ...
@coemans ?

Reply:
Ja, voor price is dit ook zo vrees ik. Daarmee dat ik dat maar heb overgenomen.

Reply:
zulke dingen (komma, punt) zijn toch internationaal bepaald, niet

Reply:
Los van de feedback van Webuniversum, denk ik dat het interessant is om de `getRawValue` van `vl.pattern.patternInstances[x].instance` via de API aan te bieden zodat er applicatief geen format/parse logica ontwikkeld moet worden.

Reply:
Ok, lijkt me op zich niet slecht om zoiets aan te bieden. 
Misschien best wel vanuit een apart issue dan?

Reply:
Prima!

Reply:
@pascalriquier  heeft het toch even voorzien op deze branch. Kijk jij even na?

Reply:
En goed voorzien he ...

Reply:
Zo heb ik ze het liefst, goed voorzien.